.ci/travis/checkchmod.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+
+CHMOD=$(find ./src ./config  -type f -perm 775)
```

Comment:
1) please find all that are not "644" it will be more strict.
2) please make certain list of file to ignore and do search in whole repo except for ".git"